src/components/DittoComponent.tsx

```diff
@@ -3,17 +3,21 @@ import { useDittoComponent } from "../hooks/useDittoComponent";
 import { DittoComponentLibraryProps } from "./Ditto";
 
 export const DittoComponent = (props: DittoComponentLibraryProps) => {
-  const { children, componentId } = props;
+  const { children, componentId, variables = {}, count } = props;
+
   const value = useDittoComponent({
     componentId,
     alwaysReturnString: typeof children !== "function",
+    variables,
+    count
   });
 
   const text = useMemo(
     () => (value !== null && typeof value === "object" ? value.text : value),
-    [value]
+    [value, variables]
   );
```

Comment:
Since `value` is already recomputed every time `variables` changes, and `variables` isn't used inside of the callback, can we remove it from this dependency array?

Reply:
👍